packages/components/src/molecules/ReviewCard/ReviewCard.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { forwardRef, useEffect, useRef, useState } from 'react'
+import type { HTMLAttributes } from 'react'
+import Rating from '../Rating'
+import Link from '../../atoms/Link'
+import ReviewAuthor from './ReviewAuthor'
+import ReviewDate from './ReviewDate'
+
+export interface ReviewCardProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * The headline of the review.
+   */
+  reviewHeadline: string
+  /**
+   * The text of the review.
+   */
+  reviewText: string
+  /**
+   * The rating of the review.
+   */
+  reviewRating: number
```

Comment:
```suggestion
  text: string
  /**
   * The rating of the review.
   */
  rating: number
```

I think we don't need to add the `review` prefix for those props

Reply:
same thing for `reviewDate`

Reply:
Done ✅